.github/workflows/e2e_ios.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   setup:
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
Can you add a comment to this file explaining why `macos-13` is required? It looks like `macos-latest` *is* `macos-13` right now. Are you just trying to avoid unpleasant surprises when it becomes `macos-14`?

Reply:
Hm, I was checking here: https://github.com/actions/runner-images/blob/main/README.md#available-images and from that it seems that macos-12 = macos-latest. macos-12 did not have Xcode 15 so we had to switch to macos-13 for this PR.